src/lexer.coffee

```diff
@@ -242,6 +242,22 @@ exports.Lexer = class Lexer
     number = match[0]
     lexedLength = number.length
 
+    # Property number notation, e.g. a.3
+    prev = @prev()
+    if prev and not prev.spaced
+      if prev[0] in [CALLABLE..., '?.', '::', '?::']
+        s = 0
```

Comment:
@zdenko in general I'd prefer more readable variable names. Makes it much easier to decipher what's going on. Eg here `s` -> `offset`, `nProp` -> `remaining` or `numChunk`, `x` -> `index`